tests/components/wemo/entity_test_helpers.py

```diff
@@ -53,7 +53,7 @@ def get_state(force_update=None):
             return
         nonlocal call_count
         call_count += 1
-        hass.add_job(waiting.set)
+        hass.loop.call_soon_threadsafe(waiting.set)
```

Comment:
`hass.add_job` will run it in the executor since `waiting.set` isn't decorated with `@callback`.

`hass.loop.call_soon_threadsafe` solves the issue as well 👍